lib/puppetlabs_spec_helper/module_spec_helper.rb

```diff
@@ -19,6 +20,7 @@ def verify_contents(subject, title, expected_lines)
 module_path = [module_path, env_module_path].join(File::PATH_SEPARATOR) if env_module_path
 
 RSpec.configure do |c|
+  c.environmentpath = spec_path
```

Comment:
Should this be
```
c.environmentpath = spec_path if Puppet.version.to_f >= 4.0
```

To preserve the same configuration for older puppet versions?